app/views/user_mailer/session_stopped_email.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p> <%= @title %> stopped streaming at <%= @time %> </p>
```

Comment:
I'd remove the whitespace after `<p>` and before `</p>` to avoid rendering them in the email